src/util/json.h

```diff
@@ -31,13 +31,22 @@ Convenience header for Nlohmann::Json that sets the default options. Also
 #include "util/Exception.h"
 #include "util/SourceLocation.h"
 
+// For higher flexibility of the custom json helper functions.
+// TODO Update with the new `SameAsAny`.
+template <typename T>
+concept IsOrderedOrUnorderedJson =
```

Comment:
I think `OrderedOrUnorderedJson` would suffice without the `is`.

Reply:
Done, see [62bc177](https://github.com/ad-freiburg/qlever/pull/1221/commits/62bc177004dd5ec9b1c494fe4800a5aacf0386e2).